internal/server/volume/server.go

```diff
@@ -181,6 +169,21 @@ func (s *Server) ResizeVolume(context context.Context, request *internal.ResizeV
 }
 
 func (s *Server) VolumeStats(context context.Context, request *internal.VolumeStatsRequest, version apiversion.Version) (*internal.VolumeStatsResponse, error) {
+	getVolumeStatsRequest := &internal.GetVolumeStatsRequest{
+		VolumeId: request.VolumeId,
+	}
+	getVolumeStatsResponse, err := s.GetVolumeStats(context, getVolumeStatsRequest, version)
+	if err != nil {
+		return nil, fmt.Errorf("Forward to GetVolumeStats failed, err=%+v", err)
+	}
+	volumeStatsResponse := &internal.VolumeStatsResponse{
+		VolumeSize:     getVolumeStatsResponse.TotalBytes,
+		VolumeUsedSize: getVolumeStatsResponse.UsedBytes,
+	}
+	return volumeStatsResponse, nil
+}
+
+func (s *Server) GetVolumeStats(context context.Context, request *internal.GetVolumeStatsRequest, version apiversion.Version) (*internal.GetVolumeStatsResponse, error) {
 	klog.V(4).Infof("calling VolumeStats with request: %+v", request)
 	minimumVersion := apiversion.NewVersionOrPanic("v1beta1")
```

Comment:
should change from v1beta1 to v1beta3 for GetVolumeStats?

Reply:
thanks for noticing this, this belongs to `VolumeStats` and not `GetVolumeStats`